main.js

```diff
@@ -1,63 +1,66 @@
-const userInput = document.querySelector('#user-input');
-const searchSection = document.querySelector('.search');
-const searchResult = document.querySelector('.search-result');
-const notificationSection = document.querySelector('.notification');
+const userInput = document.querySelector("#user-input");
+const searchSection = document.querySelector(".search");
+const searchResult = document.querySelector(".search-result");
+const notificationSection = document.querySelector(".notification");
 
 /*
-* Parse data and display them in searchResult
-*/
+ * Parse data and display them in searchResult
+ */
 const showResult = (data) => {
-    console.log(data);
-    if(data.total === 0) {
-        showNotification('Try another one or you can write it )))');
-    } else {
-        searchSection.classList.add('search-top');
-        searchResult.style.display = 'block';
-    }
-    data.data.forEach(lyrics => {
-        console.log(lyrics.title)
-        searchResult.innerHTML += `
+  console.log(data);
+  if (data.total === 0) {
+    showNotification("Try another one or you can write it )))");
+  } else {
+    searchSection.classList.add("search-top");
+    searchResult.style.display = "block";
+  }
+  data.data.forEach((lyrics) => {
+    console.log(lyrics.title);
```

Comment:
please, delete all debugging tracks (console.logs).

Reply:
Ok. Thank you. All debugging tracks are deleted.

Reply:
It is still shown.
Or you haven't pushed yet?